.github/workflows/bats.yaml

```diff
@@ -0,0 +1,17 @@
+name: Bats tests
+on: [push, pull_request]
+env:
+  functional_test: 'true'
```

Comment:
Note: I don’t know what this environment variable was for, but this variable exist in the Travis CI configuration.

Reply:
Lets remove it. This was copy pasta from another configuration and thought there might be a use case that never materialized.